Assets/Scripts/Dialogue/DialogueBubble.cs

```diff
@@ -35,52 +39,45 @@ private struct VertexAnim
 
     void Awake()
     {
+        RectTrans = GetComponent<RectTransform>();
+        Arrow = transform.parent.gameObject.transform.GetChild(1).gameObject;
+        ArrowAnim = transform.parent.GetComponentInChildren<Animator>();
         Text = GetComponentInChildren<TMP_Text>();
-        ActiveEffect = TextEffect.None;
         // Store original speed to restore after acceleration
         OriginalTextSpeed = TextSpeed;
-    }
-
-    void Start()
-    {
         Group = GetComponent<CanvasGroup>();
         Group.alpha = 0;
     }
 
     void Update()
     {
-        if (Input.GetKeyDown(KeyCode.J))
-        {
-            IsTextShaking = !IsTextShaking;
-        }
-
         // Simple hacky way to accelerate the text speed.
-        if (Input.GetButtonDown("Fast Forward")) {
-            TextSpeed = 100;
-        }
-
-        // Temporary for showcase purposes
-        if (IsTextShaking)
+        if (Input.GetButtonDown("Fast Forward"))
         {
-            StartCoroutine(ShakeText());
+            TextSpeed = FastForwardTextSpeed;
         }
```

Comment:
What if we want the text to go back to normal speed? Is that handled somewhere?

Reply:
Every time the display method is called, which is usually when a new line is displayed, the speed is assigned the normal speed